### PR TITLE
mrd-1759_GA-fix: manual fix for GA

### PR DESCRIPTION
**Pull request by bills7281**

### CODE REVIEW

File: server/views/partials/analytics.njk [hunk 1]
```diff
@@ -1,9 +1,9 @@
-  <script nonce='{{ cspNonce }}'>
-    {% if googleTagManagerId %}
-    (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-              new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-            j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-            'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-    })(window,document,'script','dataLayer','{{ googleTagManagerId }}');
-    {% endif %}
-  </script>
+<!-- Google tag (gtag.js) -->
```
[the remark this answers is not in the document]
Comment (bills7281) [reply server/views/partials/analytics.njk:1]: Yeh, do you think adding the nonce here will be required? Or would adding it potentially stop it working given the instructions on my GA account didn't specify a nonce?

[the remark this answers is not in the document]
Comment (DarrenW-HMPPS) [reply server/views/partials/analytics.njk:1]: They are probably just not getting involved with all the complexities and variations that might be required for this sort of thing.  The nonce will be something that we have added in as a security measure to identify legitimate scripts, it's not directly related to Google Tag Manager.

[the remark this answers is not in the document]
Comment (bills7281) [reply server/views/partials/analytics.njk:1]: Thanks Darren, I'vereintroduced the nonce.

[the remark this answers is not in the document]
Comment (DarrenW-HMPPS) [reply server/views/partials/analytics.njk:1]: The nonce needs to be on the inline bit of script (i.e. the second <script> tag, not the downloaded script.

[the remark this answers is not in the document]
Comment (bills7281) [reply server/views/partials/analytics.njk:1]: done

**Review by DarrenW-HMPPS — APPROVED** [review]

File: server/views/partials/analytics.njk (hunk 1 above)
[the remark this answers is not in the document]
Comment (robinandrewdevilliers) [reply server/views/partials/analytics.njk:1]: Check your console.  You will probably see an error message if the CSP is triggering.

**Pull request merged**